VideoLocker/res/values/dimens.xml

```diff
@@ -8,6 +8,10 @@
     <!-- Per Material Design Guidelines -->
     <dimen name="action_bar_icon_size">24dp</dimen>
 
+    <!-- Registration screen -->
+    <dimen name="registration_below_fields_vertical">5dp</dimen>
+    <dimen name="registration_below_fields_horizontal">2dp</dimen>
```

Comment:
Names are not that hip. How about these ?
- `registration_field_vertical_padding`
- `registration_field_horizontal_padding`